base/gfspclient/approver.go

```diff
@@ -92,3 +92,30 @@ func (s *GfSpClient) AskCreateObjectApproval(ctx context.Context, task coretask.
 	}
 	return resp.GetAllowed(), task, nil
 }
+
+func (s *GfSpClient) AskDelegateCreateObjectApproval(ctx context.Context, task coretask.ApprovalDelegateCreateObjectTask) (bool, coretask.ApprovalDelegateCreateObjectTask, error) {
+	conn, connErr := s.ApproverConn(ctx)
+	if connErr != nil {
+		log.CtxErrorw(ctx, "client failed to connect approver", "error", connErr)
+		return false, nil, ErrRPCUnknownWithDetail("client failed to connect approver, error: ", connErr)
+	}
+	req := &gfspserver.GfSpAskApprovalRequest{
+		Request: &gfspserver.GfSpAskApprovalRequest_DelegateCreateObjectApprovalTask{
+			DelegateCreateObjectApprovalTask: task.(*gfsptask.GfSpDelegateCreateObjectApprovalTask),
+		}}
+	resp, err := gfspserver.NewGfSpApprovalServiceClient(conn).GfSpAskApproval(ctx, req)
+	if err != nil {
+		log.CtxErrorw(ctx, "client failed to ask create object approval", "error", err)
```

Comment:
The logger content may be "client failed to ask delegate create object approval".